pkg/operator/controller/ingress/controller_test.go

```diff
@@ -255,11 +332,16 @@ func TestSetDefaultPublishingStrategySetsPlatformDefaults(t *testing.T) {
 		t.Run(tc.name, func(t *testing.T) {
 			ic := &operatorv1.IngressController{}
 			platformStatus := tc.platformStatus.DeepCopy()
-			if actualResult := setDefaultPublishingStrategy(ic, platformStatus, tc.domainMatchesBaseDomain); actualResult != true {
+			ingressConfig := tc.ingressConfig
+			if ingressConfig == nil {
+				ingressConfig = &configv1.Ingress{}
+			}
+			if actualResult := setDefaultPublishingStrategy(ic, platformStatus, tc.domainMatchesBaseDomain, ingressConfig); actualResult != true {
 				t.Errorf("expected result %v, got %v", true, actualResult)
 			}
 			if diff := cmp.Diff(tc.expectedIC, ic); len(diff) != 0 {
 				t.Errorf("got expected ingresscontroller: %s", diff)
+				//t.Errorf("got expected ingresscontroller: %s\n\n%+v", diff,ic.Status.EndpointPublishingStrategy.LoadBalancer)
```

Comment:
Oops, I didn't mean to include this commented code in https://github.com/miheer/cluster-ingress-operator/pull/2.

Reply:
Done.